apps/app-article-server/src/browser/ads.js

```diff
@@ -1,3 +1,6 @@
+// This file is currently unused at 2021-11-26, but could be put back into use,
```

Comment:
Can we remove this file? We have it in git history if we ever want it back (if we remember it once existed 😅)

Reply:
Yes, removed. If I use the logic in that file at all, it will be soon, so should be able to remember it's in the git history. 😅